### PR TITLE
add pricing banner

### PR DESCRIPTION
**Pull request by volodymyrmakarchuk-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-377975](https://oktainc.atlassian.net/browse/OKTA-377975)

banner

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/components/Pricing.vue [hunk 1]
```diff
@@ -61,6 +61,12 @@
           <div class="pricing--editions">
             <div class="pricing-section">
               <div class="pricing-card pricing-card-table">
+                 <div class="okta-workshop-banner">
+                    Join us at Okta21 for our Developer Talks and workshops April 6-8!
```
Comment (albertchen-okta) [inline packages/@okta/vuepress-theme-prose/components/Pricing.vue:65]: Join us at Oktane21 for our Developer Talks and Workshops,

**Review by albertchen-okta — CHANGES_REQUESTED** [review]
thanks, just some copy updates

File: packages/@okta/vuepress-theme-prose/components/Pricing.vue [hunk 2]
```diff
@@ -61,6 +61,12 @@
           <div class="pricing--editions">
             <div class="pricing-section">
               <div class="pricing-card pricing-card-table">
+                 <div class="okta-workshop-banner">
+                    Join us at Oktane21 for our Developer Talks from April 6-8, and Developer Labs on April 9!
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/components/Pricing.vue:65]:
```suggestion
                    Join us at Oktane21 for our Developer Talks April 6-8 and Developer Labs on April 9!
```
```suggestion
                    Join us at Oktane21 for our Developer Talks from April 6-8, and Developer Labs on April 9!
```

**Review by susanharper-okta — APPROVED** [review]
can I make changes to the text? If so, "from" was very misleading - can we just remove it all together and we don't really need that comma....otherwise, LGTM

**Review by albertchen-okta — APPROVED** [review]

**Review by shawyu-okta — APPROVED** [review]
Going to put up a Netlify deploy for a final validation before merging

**Comment by shawyu-okta** [comment]
https://6054e43554783c14181b78ba--reverent-murdock-829d24.netlify.app/pricing

**Comment by albertchen-okta** [comment]
netlify looks great! and I got web marketing to sign off

**Pull request merged**